node/state/core.go

```diff
@@ -1,28 +1,36 @@
 package state
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
 	"go.uber.org/fx"
 
 	"github.com/celestiaorg/celestia-app/app"
 	apptypes "github.com/celestiaorg/celestia-app/x/payment/types"
-	"github.com/celestiaorg/celestia-node/libs/keystore"
 	"github.com/celestiaorg/celestia-node/params"
 	"github.com/celestiaorg/celestia-node/service/state"
 )
 
-var keyringAccName = "default"
+var keyringAccName = "celes"
+
+func CoreAccessor(endpoint string, tp string) func(fx.Lifecycle, params.Network) (state.Accessor, error) {
+	return func(lc fx.Lifecycle, net params.Network) (state.Accessor, error) {
+		keyPath := fmt.Sprintf("~/.celestia-%s/keys", tp) // TODO @renaynay: needs to be config param
 
-func CoreAccessor(endpoint string) func(fx.Lifecycle, keystore.Keystore, params.Network) (state.Accessor, error) {
-	return func(lc fx.Lifecycle, ks keystore.Keystore, net params.Network) (state.Accessor, error) {
 		// TODO @renaynay: Include option for setting custom `userInput` parameter with
 		//  implementation of https://github.com/celestiaorg/celestia-node/issues/415.
-		ring, err := keyring.New(app.Name, keyring.BackendFile, ks.Path(), nil)
+		ring, err := keyring.New(app.Name, keyring.BackendTest, keyPath, os.Stdin)
```

Comment:
we need to eventually ensure that we use `keyring.BackendFile` right? or OS?

Reply:
Is there any way that we could simply support all the backends of the keyring? So users can chose what is best for them. I'd personally go with a ledger backend for instance.

Reply:
I could add the flag, yes.

Reply:
can we do this in a subsequent PR @liamsi ?

Reply:
yes, feel free to resolve. But keep track in an issue.